src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_30 calendar--start-day_wednesday">
+      <div class="calendar-day"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
мне кажется лучше указать через __, вроде по БЕМ так правильнее. но могу и ошибаться